packages/voice/src/models/v1/music-on-hold/music-on-hold.ts

```diff
@@ -0,0 +1 @@
+export type MusicOnHold = 'ring' | 'music1' | 'music2' | 'music3';
```

Comment:
Why sometime a type is defined for supported enum values like here with  `MusicOnHold` or `TtsVoice` and sometime the authorized values are directly set at "usage" level  (see the PR https://github.com/sinch/sinch-sdk-node/pull/24/files#diff-310899169f21ebf8d587f23f4b09b4e15353f657ff6b7dd7a705807229ab0e73R8) ?

Reply:
It depends 😅
Generally, when the type is super simple (one or two values), I define it inline so it's still readable (this is the case for all the actions and instructions names for instance)
When the type is used several times, I prefer to create a dedicated one, such as `MusicOnHold` which is used a couple of times
And when the type is super complex (like the supported locales or the conversation triggers), it deserves it's own type.
But writing these lines, I realized that I'm not consistent: sometimes I create a dedicated file, sometimes I group the enum types in the `enum.ts` file. I'll have to refactor this!

Reply:
Fixed